scripts/run_examples_dev

```diff
@@ -7,38 +7,56 @@ source "$SCRIPTS_DIR/common"
 # Build Oak server.
 "$SCRIPTS_DIR/build_server_dev" "$@"
 
-# Run Oak server.
-./bazel-clang-bin/oak/server/dev/oak &
-SERVER_PID=$!
 # TODO: Ensure that background processes are killed with something like `trap cleanup_fn EXIT`.
 
 # Run Rust-based Oak examples against a common Oak server instance.
-# Note that the chat example cannot be run because the client is interactive.
-# TODO(#462): Re-enable chat example, even if it may be just loading the server-side code and not
-# interact with it.
 # TODO(#594): Re-enable rustfmt when upstream rustc internal error is fixed.
-readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq | grep -v chat | grep -v rustfmt)"
+readonly RUST_EXAMPLES="$(find examples -mindepth 2 -maxdepth 4 -type d -regex '.*/module.*/rust$' | cut -d'/' -f2 | uniq | grep -v rustfmt)"
 for example in ${RUST_EXAMPLES}; do
-  "${SCRIPTS_DIR}/run_example" "${example}"
+  # Running `build_example` to compile application configurations before running the server.
+  "${SCRIPTS_DIR}/build_example" "${example}"
+  ./bazel-clang-bin/oak/server/dev/dev_oak_runner \
+    --config="$PWD/bazel-client-bin/examples/${example}/config/config.bin" &
+  SERVER_PID=$!
+  sleep 2  # Wait for an application to start.
```

Comment:
Oh my.. there must be a better way :)

Reply:
I think it will require rewriting this script in Python, so we could parse the output of the `oak_runner` in order to understand that the module was loaded.